packages/design-system-twrnc-preset/src/typography.ts

```diff
@@ -5,71 +5,71 @@ import type { TypographyTailwindConfigProps } from './typography.types';
 export const typographyTailwindConfig: TypographyTailwindConfigProps = {
   fontSize: {
     'display-lg': [
-      typography.sDisplayLG.fontSize,
+      typography.sDisplayLG.fontSize.toString(),
```

Comment:
Updating typography types to adhere to the TwConfig from `import type { TwConfig } from 'twrnc';` in tailwind.config.ts